modules/refine-workflow/src/test/java/org/openrefine/exporters/OdsExporterTests.java

```diff
@@ -115,7 +114,7 @@ public void exportSimpleOds() throws IOException {
 
         try {
             OdfDocument odfDoc = OdfDocument.loadDocument(new ByteArrayInputStream(stream.toByteArray()));
-            List<OdfTable> tables = odfDoc.getTableList();
+            List<OdfTable> tables = odfDoc.getTableList(true);
```

Comment:
We usually try to avoid mixing unrelated changes together, so we'd normally want this deprecation update in a separate PR.